### ISSUE TITLE
Memory leak using .table() ?

### ISSUE THREAD

**Issue by mrorigo** [cronvel/terminal-kit#138]
This seems to be leaking memory, it blows up pretty quickly with out of memory error.

```
const term = require('terminal-kit').terminal;

const rows = 3, cols = 100;
const labels = (new Array(cols)).fill(0).map((x, i) => i);
const data = [labels].concat((new Array(rows).fill(0)).map(x => (new Array(cols).fill(1))));

term.fullscreen(true);

while (true) {
    term.moveTo(1, 1);
    term.table(data, {});
}
```

**Comment by cronvel** [comment]
Thanks for reporting, I don't know what can cause this at the moment.

**Comment by cronvel** [comment]
It's very hard to detect that leak... I published v1.44.1 that mitigate it, it still leaks but probably hundred of times slower.

**Comment by cronvel** [comment]
It's fixed now as of v1.44.3, thanks for reporting it ;)
